tiledb/query.py

```diff
@@ -14,20 +19,272 @@ def __init__(
         self,
         array: Array,
         ctx: Ctx = None,
+        attrs=None,
+        cond=None,
+        dims=None,
+        coords=False,
+        index_col=True,
+        order=None,
+        use_arrow=None,
+        return_arrow=False,
+        return_incomplete=False,
```

Comment:
I think in general if we can set a specific default value for the argument here and avoid `None` that it would make the code easier to follow and less verbose.

Reply:
Yes, did that. Good idea.